### DISCUSSION TITLE
GraphScope is now on our client's production environment as a Graph Computing service

### DISCUSSION THREAD

**Discussion by TerryWang1208** [alibaba/GraphScope#2500] [Show and tell]
Hi guys,

We are from Fabarta Inc. - a Graph Intelligience company.

Glad to share with you that we have successfully released our graph computing service, which is based on GraphScope, to one of our clients in the financial industry. The service contained 4 nodes(8c 32g for each), serving graph computing with 20,000,000 vertices and 100,000,000 edges.

Thanks to the good documents and great support from the community here(especially @sighingnow ), the way to bring the powerful graphscope to production was smooth. Moreover, we have implemented a tiKV loader as a customization of GraphScope to make it possible to feed graphscope with graph data from a tiKV cluster.

The version we focused on is 0.17, and we are working on the upgrade to the latest version where we have seen a big improvement on the memory peak usage during data loading. 

Will keep you updated on how we use GraphScope in the real world.

**Comment by sighingnow** [comment]
Thanks a lot for sharing!

We feel proud that graphscope can help in real-world applications and make things better!
